src/gpu/GPU.h

```diff
@@ -0,0 +1,10 @@
+#include "../timestamp.hpp"
+
+#define MAX_MARKERS_PER_READ (1 << 15)
+#define GPU_BATCH_SIZE (1 << 12)
+#define MIN_GPU_BATCH_SIZE (1 << 4)
+
+void common_markers (uint64_t* read_pairs, uint32_t offset, int n);
```

Comment:
Is it possible to move this to the shasta namespace? I want to keep additions to the global namespace to a minimum to reduce the potential for name collisions and also for clarity.

Reply:
I am removing this function as I do not use it anymore.